### PR TITLE
pkg/ansi: fix cursor save/restore

### PR DESCRIPTION
**Pull request by eandre**
It turns out the terminal escape codes we used were not
supported by all terminals, notably macOS's Terminal.app.

Fix this by using a different terminal escape code that does work.

### CODE REVIEW

File: e2e-tests/testdata/echo/go.mod [hunk 1]
```diff
@@ -2,7 +2,6 @@ module encore.app
 
 go 1.18
 
-require "encore.dev" v1.1.0
-
-replace encore.dev => ../../../../../runtime
+require "encore.dev" v1.9.3
 
+replace encore.dev => ../../../runtime
```
Comment (eandre) [inline e2e-tests/testdata/echo/go.mod:7]: This is just unrelated cleanup, the replace directive needed to be updated since we moved the `e2e-tests` directory

**Review by DomBlack — APPROVED** [review]
